app/main.py

```diff
@@ -1,3 +1,28 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.customer import Customer
+from app.car import Car
+
+
+def shop_trip() -> None:
+    with open(r"app\config.json") as file:
```

Comment:
The path to the JSON file should use forward slashes (/) instead of backslashes (\) to ensure compatibility across different operating systems. According to the checklist, make sure you write the correct path to the JSON file.